Sources/Widgets/Representations/ArrowHandleRepresentation/index.js

```diff
@@ -125,8 +168,52 @@ function vtkArrowHandleRepresentation(publicAPI, model) {
 
   // --------------------------------------------------------------------------
 
-  publicAPI.requestData = (inData, outData) => {
-    const { points, scale, color } = model.internalArrays;
+  function handleGlyphRotation(scale3) {
+    const viewMatrix = mat3.create();
+    mat3.fromMat4(viewMatrix, model.viewMatrix);
+    const viewMatrixInv = mat3.create();
+    mat3.invert(viewMatrixInv, viewMatrix);
+
+    let displayOrientation = vec3.create();
+    let baseDir = [0, 1, 0];
+    switch (model.currShape) {
+      case HandleRepresentationType.CONE:
+        displayOrientation = [0, 1, 0];
+        break;
+      case HandleRepresentationType.CIRCLE:
+        displayOrientation = [0, 0, 1];
```

Comment:
why do you bother with displayorientation for a circle ? can't you simplify it to be the same than the cone ?

Reply:
The circle source is oriented not to face camera, so i had to reorient it. See CircleSource example to check classic orientation (you have to drag the source to see orientation is on the Y axis)

Reply:
Maybe you should expose an orientation parameter to vtkCircleSource.